src/database/models/location.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+import { Model } from 'sequelize';
+export default (sequelize, DataTypes) => {
+  class Location extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate({ accomodation }) {
+      this.hasMany(accomodation, {
+        foreignKey: 'locationId',
```

Comment:
I am able to delete location while there are accomodations depends on it (accomodation still exist with unavailble location).
is it how you intended to do it or some wrong with association?